boards/particle-argon/Makefile.include

```diff
@@ -1,3 +1,4 @@
 USEMODULE += boards_common_particle_mesh
+USEMODULE += periph_uart_hw_fc
```

Comment:
It would be better to add this in Makefile.dep

Reply:
done.

Reply:
It seems that this comment is still unaddressed. Those 2 USEMODULE should be in Makefile.dep.

Reply:
Must be due to the rebase... I knew I shouldn't rebase at night...